atomic_defi_design/qml/Settings/SettingModal.qml

```diff
@@ -156,6 +156,21 @@ Qaterial.Dialog {
                                     onCheckedChanged: API.app.settings_pg.notification_enabled = checked
                                 }
                             }
+                            RowLayout {
+                                width: parent.width-30
+                                anchors.horizontalCenter: parent.horizontalCenter
+                                height: 50
+                                DexLabel {
+                                    Layout.alignment: Qt.AlignVCenter
+                                    Layout.fillWidth: true
+                                    text: qsTr("Maximum number of enabled coins")
+                                }
+                                DexComboBox {
+                                    model: [10, 20, 50, 75, 100, 150, 200]

```

Comment:
Here it's should be possible to detect the maximum value, which is the global cfg size i believe